docs/API.md

```diff
@@ -481,6 +481,29 @@ formatted as IPv4-mapped IPv6 addresses). By default (-1) this option is not set
 and the platform default value is used.
 ---
 
+| OptName               | Since | Binding | Type      | Units  | Default  | Range  |
+| --------------------- | ----- | ------- | --------- | ------ | -------- | ------ |
+| `SRTO_KMREFRESHRATE`  | 1.3.2 | pre     | `int32_t` | pkts   | 0x1000000| 0..unlimited |
+
+- **[GET or SET]** - number of packets to be transmitted after which the encryption
+key will be switched to the new one. The process involves "preannounce" when the new
```

Comment:
will be switched to the new one. The switchover occurs in two steps: first, there is a "preannounce" period during which the new

Reply:
I still changed it a little bit. :) Here is the full text for SRTO_KMREFRESHRATE, please review before I commit again.

- **[GET or SET]** - The number of packets to be transmitted after which the Stream
Encryption Key (SEK), used to encrypt packets, will be switched to the new one.
Note that the old and new keys live in parallel for a certain period of time
(see `SRTO_KMPREANNOUNCE`) before and after the switchover, named "preannounce"
and "decommission" respectively. The "preannounce" period is required so that
the new SEK is installed in the receiver before the first packet encrypted with
the new SEK is received, and the old key should remain active after switchover
in order to decrypt packets being in flight at that time and packets to be 
retransmitted.

Reply:
```
- **[GET or SET]** - The number of packets to be transmitted after which the Stream
Encryption Key (SEK), used to encrypt packets, will be switched to a new one.
Note that the old and new keys live in parallel for a certain period of time
(see `SRTO_KMPREANNOUNCE`) before and after the switchover.
<br><br>
Having a preannounce period before switchover ensures the new SEK is installed at the receiver before the first packet encrypted with the new SEK is received. The old key remains active after switchover in order to decrypt packets that might still be in flight, or packets that have to be retransmitted. 
```